packages/datadog-instrumentations/src/opensearch.js

```diff
@@ -4,7 +4,7 @@ const { addHook } = require('./helpers/instrument')
 const shimmer = require('../../datadog-shimmer')
 const { createWrapRequest } = require('./elasticsearch')
 
-addHook({ name: '@opensearch-project/opensearch', file: 'lib/Transport.js', versions: ['1.1.0'] }, Transport => {
+addHook({ name: '@opensearch-project/opensearch', file: 'lib/Transport.js', versions: ['1.x', '2.x'] }, Transport => {
```

Comment:
Why not just use `>=1`? It would support new versions automatically as other instrumentations do.

Reply:
Done